debian/control

```diff
@@ -0,0 +1,51 @@
+Source: paflib
+Section: libs
+Maintainer: Carlos Eduardo Seo (ceseo) <contact@carlosseo.com>
```

Comment:
It would be better to mention the authors listed in AC_INIT (in the file configure.ac).

Currently, configure.ac looks like this:

    AC_INIT([libpaf], [MAJOR.MINOR.PATCH], [tulioqm@br.ibm.com emachado@br.ibm.com])

It lists two author, however, according to debian policy, "the maintainer may be one person or a group of people reachable from a common email address, such as a mailing list" [1].

Since we do **not** have a group email, I suggest that you change this line to:

    Maintainer: Tulio Magno Quites Machado Filho <tulioqm@br.ibm.com>


[1] https://www.debian.org/doc/debian-policy/ch-binary.html#s-maintainer